polaris/ocean/model/ocean_model_step.py

```diff
@@ -84,20 +83,25 @@ def __init__(self, component, name, subdir=None, indir=None, ntasks=None,
             If so, the partitioning executable is taken from the ``partition``
             option of the ``[executables]`` config section.
 
-        graph_filename : str, optional
-            The name of the graph file to partition
+        graph_target : str, optional
+            The graph file name (relative to the base work directory).
+            If none, it will be created.
         """
+        if graph_target is None:
+            self.make_graph = True
+
         super().__init__(
             component=component, name=name, subdir=subdir, indir=indir,
             ntasks=ntasks, min_tasks=min_tasks, openmp_threads=openmp_threads,
             max_memory=max_memory, cached=cached, yaml=yaml,
             update_pio=update_pio, make_graph=make_graph,
             mesh_filename=mesh_filename, partition_graph=partition_graph,
-            graph_filename=graph_filename)
+            graph_filename='graph.info')
 
         self.dynamic_ntasks = (ntasks is None and min_tasks is None)
 
         self.map: Union[None, List[Dict[str, Dict[str, str]]]] = None
+        self.graph_target = graph_target
```

Comment:
To be thorough, I think we should add this as an attribute in the docstring for the class above.

Reply:
Thanks for catching that